README.md

```diff
@@ -26,11 +26,11 @@ This documentation is up to date as of vrecord version 2017-07-15.
 
 ## Summary
 
-Vrecord is open-source software for capturing a video signal and turning it into a digital file. Its purpose is to make videotape digitization or transfer easier. Vrecord can capture analog and digital signals through a variety of inputs and can create digital video files in a variety of formats and codecs. Vrecord has been designed with needs of audiovisual archivists in mind. 
+Vrecord is open-source software for capturing a video signal and turning it into a digital file. Its purpose is to make videotape digitization or transfer easier. Vrecord can capture analog and digital signals through a variety of inputs and can create digital video files in a variety of formats and codecs. Vrecord has been designed with needs of audiovisual archivists in mind.
```

Comment:
nitpick, i'd suggest to having spacing changes in separate commits than content changes.

Reply:
sorry, have been flustered trying to get Atom settings to stop automatically eliminating whitespace. new commit without spacing changes below